test/test_littlefs.c

```diff
@@ -23,6 +23,11 @@
 #include "esp_partition.h"
 #include "errno.h"
 
+#if __has_include("esp_rom_sys.h")
+#include "esp_rom_sys.h"
+#define ets_printf esp_rom_printf
+#endif
```

Comment:
Similar to the comment above, esp_rom_printf is available since v4.3. It might be a bit less technical debt left if you change ets_printf to esp_rom_printf in this file, and then leave a `#define esp_rom_printf ets_printf` for older versions.

Reply:
addressed in [ddbe244](https://github.com/joltwallet/esp_littlefs/pull/91/commits/ddbe244b50ee9595bfd314d286e2db3117274149)